meson.build

```diff
@@ -64,6 +64,7 @@ common_SOURCES = ['src/logger.cpp',
                   'src/ipz_parser.cpp',
                   'src/keyword_vpd_parser.cpp',
                   'src/ddimm_parser.cpp',
+                  'src/isdimm_vpd_parser.cpp',
```

Comment:
could you rename the file to indium_parser.cpp please

Reply:
Done.